src/annotate/celltypist/config.vsh.yaml

```diff
@@ -0,0 +1,114 @@
+functionality:
+  name: celltypist
+  namespace: annotate
+  description: Automated cell type annotation tool for scRNA-seq datasets on the basis of logistic regression classifiers optimised by the stochastic gradient descent algorithm.
+  authors:
+    - __merge__: /src/authors/jakub_majercik.yaml
+      roles: [ author ]
+
+  argument_groups:
+    - name: Inputs
+      description: Arguments related to the input (aka query) dataset.
+      arguments:
+        - name: "--input"
+          alternatives: [-i]
+          type: file
+          description: Input h5mu file.
+          direction: input
+          required: true
+          example: input.h5mu
+        - name: "--modality"
+          description: Which modality to process.
+          type: string
+          default: "rna"
+          required: false
+        - name: "--input_obs_label"
+          type: string
+          description: Key in obs field of input adata for label information.
+          required: false
+    - name: Reference
+      description: Arguments related to the reference dataset.
+      arguments:
+        - name: "--reference"
+          type: file
+          description: "User-provided reference dataset. The data that will be used as reference to call cell types."
+          example: reference.h5ad
+          direction: input
+          required: false
+        - name: "--reference_obs_label"
+          type: string
+          description: Key in obs field of reference AnnData with cell-type information.
```

Comment:
```suggestion
          multiple: true
          description: "The `.obs` key(s) of the target labels to tranfer."
```

Reply:
Some reference atlases have different columns of labels, to allow for different granularity of cell type labels, could you adjust the script as well to allow for multiple columns? (see `src/labels_transfer/knn`)